pandas/io/excel/_base.py

```diff
@@ -897,7 +885,7 @@ def parse(
 
                 output[asheetname] = parser.read(nrows=nrows)
 
-                if not squeeze or isinstance(output[asheetname], DataFrame):
+                if isinstance(output[asheetname], DataFrame):
```

Comment:
just for my understanding - previously, `isinstance(output[asheetname], DataFrame)` would only have been evaluated if `squeeze` was `True`. Now that that option is `False` always (because `squeeze` has been removed), then can this whole `if` statement be removed?

Reply:
good point, thx. Yes we always get back a DataFrame, so can remove